api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -0,0 +1,294 @@
+package org.pmiops.workbench.db.jdbc;
+
+import static org.pmiops.workbench.db.model.DbStorageEnums.billingAccountTypeFromStorage;
+import static org.pmiops.workbench.db.model.DbStorageEnums.billingStatusFromStorage;
+import static org.pmiops.workbench.db.model.DbStorageEnums.dataAccessLevelFromStorage;
+import static org.pmiops.workbench.db.model.DbStorageEnums.institutionDUATypeFromStorage;
+import static org.pmiops.workbench.db.model.DbStorageEnums.organizationTypeFromStorage;
+import static org.pmiops.workbench.utils.mappers.CommonMappers.offsetDateTimeUtc;
+
+import java.util.List;
+import org.pmiops.workbench.model.ReportingCohort;
+import org.pmiops.workbench.model.ReportingDataset;
+import org.pmiops.workbench.model.ReportingDatasetCohort;
+import org.pmiops.workbench.model.ReportingDatasetConceptSet;
+import org.pmiops.workbench.model.ReportingDatasetDomainIdValue;
+import org.pmiops.workbench.model.ReportingInstitution;
+import org.pmiops.workbench.model.ReportingUser;
+import org.pmiops.workbench.model.ReportingWorkspace;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReportingQueryServiceImpl implements ReportingQueryService {
+
+  private final JdbcTemplate jdbcTemplate;
+
+  public ReportingQueryServiceImpl(JdbcTemplate jdbcTemplate) {
+    this.jdbcTemplate = jdbcTemplate;
+  }
+
+  @Override
+  public List<ReportingWorkspace> getWorkspaces() {
+    return jdbcTemplate.query(
+        "SELECT \n"
+            + "  billing_account_type,\n"
```

Comment:
Since you hardcode the column names below anyways - is there a reason not to just do a `select *` here?

Reply:
The Ruby stuff is hardened against places things need to be renamed for some reason. If it's not needed anymore that would be an OK simplification.

At HubSpot we had a human SQL linter who would watch hundreds of repos for SQL changes via some kind of script and jump in to tell us things. One of those was never to use `*`, as the semantics of your queries could change on you when someone adds or removes or changes a column. </war_story>